typeSummary.txt

```diff
@@ -48256,9 +48389,9 @@ class Microsoft.Graph.UserRevokeSignInSessionsRequest : Microsoft.Graph.BaseRequ
     return type Microsoft.Graph.IUserRevokeSignInSessionsRequest
     param value : System.String
   method PostAsync
-    return type System.Threading.Tasks.Task`1[[System.Boolean]]
+    return type System.Threading.Tasks.Task`1[[System.Nullable`1[[System.Boolean]]]]
```

Comment:
@MIchaelMainer 
Sorry for not catching this earlier, but this is a breaking change. Unfortunately this is already in but is reflected in type summary because previous check-in did not generate the type summary, i.e. the commit was not generated using the pipeline.

See the linqpad example below for how people's code can break:

```
void Main()
{
	var output = GetAsync().Result;
	var output2 = GetAsync2().Result;
	if (output)
	{
		Console.WriteLine("Compiles");
	}
	
	if (output2)
	{
		Console.WriteLine("Does not compile because output2 is a nullable boolean and can't be used in if conditions");
	}
}

public async Task<bool> GetAsync()
{
	return true;
}

public async Task<bool?> GetAsync2()
{
	return true;
}
```

Reply:
@zengin You are technically correct. The thing is that it was already broken and wasn't working before. So no one could've taken a dependency on these features. So this change does make it not breaking. The issue for me is that how did this not get in the manual build?

Reply:
Thanks for the clarification on the breaking change.

Did you generate #800 using the pipeline? If not, it is expected that the type summary was not updated.

Reply:
No, I didn't. It would've be useful for me to capture summary in my PR.

Reply:
@baywet @nikithauc @MIchaelMainer 

Btw, I run into these non-breaking breaking changes every now and then. Can we come up (or reuse) some term for this so that I can just say `bla?` and `bla?` represents:

`This is technically a breaking change, but is this one of those cases where nobody could've taken a dependency on this due to broken state so it is not a breaking change?`

Does anybody know such a term?
cc: @ddyett @darrelmiller @peombwa